### ISSUE TITLE
[Bug]: PDI Don't get the same result using temporalBehaviour and spatialBheaviour

### ISSUE THREAD

**Issue by thomaspibanez** [umr-amap/StormR#60]
### What happened?

Trying to get PDI for a plot in New Caledonia after Niran and found very different results. Values obtained with spatialBheaviour are almost 1000 times higher than those obtain directly with temporalBheaviour?

### Minimal Complete Verifiable Example

```R
IBTrACS<-defDatabase("IBTrACS.ALL.v04r00.nc",verbose=FALSE)

st<-Storms(IBTrACS,loi=c(166.9039,-22.27495),seasons=2021,names="NIRAN",verbose=0)
pts<-data.frame(x=166.9039,y=-22.27495)

#Using temporalBehaviour()
PDIt<-temporalBehaviour(st,pts,product=("PDI"),verbose=0)
PDIt # 0.328

#Using spatialBehaviour()
PDI_NC<-spatialBehaviour(st,product="PDI")
PDIs<-terra::extract(PDI_NC,cbind(166.9039,-22.27495))
PDIs # 214.766


#Using temporalBehaviour()
TS<-temporalBehaviour(st,pts,product=("TS"),verbose=0)
MSWt<-max(TS[[1]][[1]]$speed)
MSWt # 30.461

#Using spatialBehaviour()
MSW_NC<-spatialBehaviour(st,product="MSW")
MSWs<-terra::extract(MSW_NC,cbind(166.9039,-22.27495))
MSWs # 30.52791
```

### Environment

<details>

</details>

**Comment by BaptisteDlp** [comment]
Fix in commit 6e458b31de0f1f553f5bd2237e73f289eaaaff64

**Comment by BaptisteDlp** [comment]
202.218 for temporalBehaviour and 214.766 with spatialBehvaiour. Differences come from the `terra::focal(mean)` used in spatialBehaviour
